src/aspire/utils/rotation.py

```diff
@@ -234,10 +234,29 @@ def from_euler(values, dtype=np.float32):
         :param values: Rotation angles in radians, as a n x 3 array
         :return: new Rotation object
         """
-        rotations = sp_rot.from_euler("ZYZ", values.astype(dtype), degrees=False)
+        rotations = sp_rot.from_euler("ZYZ", values, degrees=False)
         matrices = rotations.as_matrix().astype(dtype)
         return Rotation(matrices)
 
+    @staticmethod
+    def about_axis(axis, angles, dtype=np.float32):
+        """
+        Build rotation object from axis and angles of rotation.
+
+        :param axis: A string denoting the axis of rotation. "x", "y", or "z".
+        :param angles: Rotation angles in radians.
+        :param dtype: Data type for rotation matrices.
+
+        :return: Rotation object
+        """
+        axes = ["x", "y", "z"]
+        if axis.lower() not in axes:
+            raise AssertionError("Axis must be 'x', 'y', or 'z'")
```

Comment:
I think this should be a `ValueError` unless you were to do

```python
assert axis.lower() in axes, "Axis must be in 'x', 'y', or 'z'"
```

which would raise `AssertionError` on its own.